qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -37,6 +37,12 @@
             filesize = 100
             unplug_chardev_vc1 = no
             unplug_chardev_vc2 = no
+        - unplug_port_chardev_pci:
+            type = virtio_serial_unplug_port_chardev_pci
+            sleep_time = 0
+            start_vm = yes
```

Comment:
No need, it is the default value in framwork

Reply:
thanks and update